package.json

```diff
@@ -24,6 +24,7 @@
     "d3-scale": "^3.2.1",
     "d3-shape": "^1.3.7",
     "d3-time": "^2.0.0",
+    "html-entities": "^1.3.1",
```

Comment:
any `package.json` needs to have pairing `yarn.lock` update. make sure to include it.